platform/commonUI/general/res/sass/_archetypes.scss

```diff
@@ -97,23 +97,54 @@
 .l-flex-col {
     @include display-flex;
     @include flex-wrap(nowrap);
+    //@include flex(1 1 auto !important);
     .flex-elem {
+        //@include test(green, 0.1);
```

Comment:
Is there value/utility to having these commented-out lines here? In general prefer to remove unused code